src/test/scala/chiselTests/ProbeSpec.scala

```diff
@@ -733,4 +733,13 @@ class ProbeSpec extends ChiselFlatSpec with MatchesAndOmits with Utils {
       "output b : Probe<UInt<2>, LayerA.LayerB>"
     )()
   }
+
+  "Probes" should "have valid names" in {
+    class TestMod extends RawModule {
+      val a = IO(Output(Probe(UInt())))
+      val w = WireInit(UInt(32.W), 0.U)
+      a := w
+      require(reflect.DataMirror.queryNameGuess(a) == "a")
+    }
```

Comment:
```suggestion
    class TestMod extends RawModule {
      val a = IO(Output(Probe(UInt())))
      val w = WireInit(UInt(32.W), 0.U)
      a := w
      require(reflect.DataMirror.queryNameGuess(a) == "a")
    }
    ChiselStage.emitCHIRRTL(new TestMod)
```
You have to actually run elaboration for this test to do anything. This is why I always make sure my unit tests fail before they pass (e.g. change the check to say it's equal to `"b"`, see it fail, then fix it).